### ISSUE TITLE
PREPROD - Liste des départements dans la recherche est en dur

### ISSUE THREAD

**Issue by syji35** [megalis-bretagne/focus-marches#14]
Sur la page recherche, en mode avancé, la liste des département est positionné en dur.

@lvinsonneau @ElodieTessier 
Proposition : Dynamiser la liste des départements sur la page de recherche en mode avancé en incluant les 10 départements les plus représentés dans les marchés.

**Comment by lvinsonneau** [comment]
Dans l'idéal, ce serait bien d'en discuter dans la communauté Focus marchés élargie (cf. ARNia + RECIA), mais dans l'urgence de corriger ce qui est du ressort du bug chez nous (puisqu'on a une rubrique vide), je me dis que ta proposition n'est pas déconnante.
Vos avis @spelhate @barbitruc @ElodieTessier  ?

**Comment by syji35** [comment]
Correction apporté. 
Déployé en preprod sur prochaine release

**Comment by ElodieTessier** [comment]
Je suis d'accord avec @lvinsonneau, pour les évolutions qui vont impacter tout Focus Marchés. Autant en échanger ensemble pour se mettre d'accord sur la meilleure réponse à apporter, qui conviendra à la majorité et qui sera développée une seule fois par et pour la communauté. 
Maintenant, la proposition me semble intéressante. Par contre, si qqu'un déploie Focus marchés pour l'ensemble de la France entière ou une région avec plus de 10 départements, c'est gênant. Pour la Bretagne c'est bon mais pour la Nouvelle Aquitaine avec ses 12 départements par exemple ce n'est pas suffisant. C'est une réflexion, je n'ai pas de réponse définitive sur le sujet.
